pages/staff.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+	<main>
+		<navbar />
+
+		<h1>Staff</h1>
+		<p>Our fantastic team members</p>
+
+		<div class="container" id="staff">
+			<card id="Aimbot">
```

Comment:
Instead of hardcoding each user, put the data into an array in the script tag and loop over it, with the template v-for in vue.

Reply:
Will be changed in a later PR